src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Create/Endpoint.cs

```diff
@@ -21,9 +21,10 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
             .AsTracking()
             .FirstOrDefaultAsync(x => x.ResourceContentVersionId == req.ResourceContentVersionId && x.ContentIndex == req.ContentIndex, ct);
 
-        if (existingMt is not null)
+        if (existingMt is not null && !(existingMt.Created > DateTime.UtcNow.AddMinutes(-60)))
```

Comment:
Nit: `.AddHours(-1)`

Reply:
I think we also need a check that `existingMt.RetranslationReason is not null`, because they can only do it once.

Reply:
And really, if that isn't null, then `request.RetranslationReason` can't be empty or null.